components/commons/basebutton.css

```diff
@@ -11,7 +11,6 @@ governing permissions and limitations under the License.
 */
```

Comment:
Note that this is the basebutton-coretokens.css file that has been moved to basebutton.css and that no changes were made to the actual commons assets.